cmd/web3/start.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	"github.com/gochain-io/web3"
+	"github.com/urfave/cli"
+)
+
+func start(c *cli.Context) error {
+
+	privateKey := os.Getenv("WEB3_PRIVATE_KEY")
```

Comment:
The other cases pull this through the cli package API as a flag with corresponding env var